### PR TITLE
[WFLY-10475] JSF installer module definitions are not synced with JSF…

### PR DESCRIPTION
**Pull request by treblereel**
JIRA: https://issues.jboss.org/browse/WFLY-10475

### CODE REVIEW

**Comment by kabir** [comment]
@fjuma Please approve

File: jsf/multi-jsf-installer/src/main/resources/mojarra-injection-module.xml [hunk 1]
```diff
@@ -35,14 +35,19 @@
 
     <dependencies>
         <module name="com.sun.jsf-impl" slot="${jsf-impl-name}-${jsf-version}"/>
-        <module name="javax.api"/>
+        <module name="java.naming"/>
+        <module name="java.desktop"/>
+        <module name="org.jboss.as.ee"/>
```
Comment (fjuma) [inline jsf/multi-jsf-installer/src/main/resources/mojarra-injection-module.xml]: The "org.jboss.as.ee" dependency is already listed below.

File: jsf/multi-jsf-installer/src/main/resources/myfaces-injection-module.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.5" name="org.jboss.as.jsf-injection" slot="${jsf-impl-name}-${jsf-version}">
+<module xmlns="urn:jboss:module:1.8" name="org.jboss.as.jsf-injection" slot="${jsf-impl-name}-${jsf-version}">
```
Comment (fjuma) [inline jsf/multi-jsf-installer/src/main/resources/myfaces-injection-module.xml]: Dependencies on the "org.jboss.as.jsf" module and on the "org.jboss.weld.api" module should be added to the dependencies in this file as well.

File: jsf/multi-jsf-installer/src/main/resources/mojarra-injection-module.xml (hunk 1 above)
Comment (fjuma) [inline jsf/multi-jsf-installer/src/main/resources/mojarra-injection-module.xml]: The "org.jboss.as.ee" module dependency is already listed below.

**Review by fjuma — CHANGES_REQUESTED** [review]

**Review by fjuma — APPROVED** [review]
Thanks @treblereel!

**Comment by erickdeoliveiraleal** [comment]
there's some differences in module.xml between myfaces and mojarra.
eg.:  <module name="org.jboss.weld.core"/>
        <module name="org.jboss.weld.spi"/>
        <module name="javax.faces.api"/>
is it right? when wildfly starts it complains about.

> Using deployment classloader to load proxy classes for module com.sun.jsf-impl:myfaces-2.3.1. Package-private access will not work. To fix this the module should declare dependencies on [org.jboss.weld.core, org.jboss.weld.spi]

**Pull request merged**